### DISCUSSION TITLE
Corrupted quiz link

### DISCUSSION THREAD

**Discussion by EGalahad** [microsoft/ML-For-Beginners#604] [Q&A]
The pre- and post-lecture quiz link is corrupted, 404 not found.

**Comment by jlooper** [comment]
@softchris @leestott all the quizzes are down and should be relaunched as ASWA

**Reply by hunter5309** [reply]
👍

**Comment by Queen3Bella** [comment]
Anyone found a way to access the pre- and post quizzes?

**Comment by Shubham-Modi** [comment]
Quiz links are still not working even on ASWA.
